litellm/proxy/proxy_cli.py

```diff
@@ -502,6 +510,16 @@ def _make_openai_completion():
                     ssl_keyfile=ssl_keyfile_path,
                     ssl_certfile=ssl_certfile_path,
                 )  # run uvicorn
+            elif ssl_ca_bundle_path is not None:
+                print(
+                    f"\033[1;32mLiteLLM Proxy: Using SSL with CA Bundle: {ssl_ca_bundle_path}\033[0m\n"
+                )
+                uvicorn.run(
+                    app,
+                    host=host,
+                    port=port,
+                    ssl_ca_certs=ssl_ca_bundle_path,
+                )  # run uvicorn
```

Comment:
this looks good to me, just need to check uvicorn supports `ssl_ca_certs` and gunicorn supports `ca_certs` 

@nivibilla can you confirm the proxy works on your side with these changes ?